services/web/server/src/simcore_service_webserver/.oas3/v1/openapi.yaml

```diff
@@ -0,0 +1,215 @@
+        E.g. to confirm account, password reset, etc. Token encrypts user and
+        action to be confirmed
+      parameters:
+        - name: token
+          in: path
+          description: Confirmation token encoding idenfitier and operation to confirm
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        default:
+          description: Default error
+          content:
+            application/json:
+              schema:
+                $ref: './errors.yaml#/components/schemas/ErrorEnveloped'
+  /login:
+    post:
+      tags:
+        - authentication
+      summary: Logs client into the service
+      operationId: login
+      responses:
+        default:
+          description: Default error
+          content:
+            application/json:
+              schema:
+                $ref: './errors.yaml#/components/schemas/ErrorEnveloped'
+  /logout:
+    post:
+      tags:
+        - authentication
+      summary: Logs out client from the service
+      operationId: logout
+      responses:
+        default:
+          description: Default error
+          content:
+            application/json:
+              schema:
+                $ref: './errors.yaml#/components/schemas/ErrorEnveloped'
+#-------------------------------------------------------
+components:
+
+  schemas:
+    HealthCheckEnveloped:
+      type: object
+      properties:
+        data:
+          $ref: '#/components/schemas/HealthCheck'
+        status:
+          type: integer
+          format: int32
+          example: 200
+    HealthCheck:
+      type: object
+      properties:
+        name:
+          type: string
+          example: simcore-service-webserver
+        status:
+          type: string
+          example: SERVICE_RUNNING
+        version:
+          type: string
+          example: 2.0.1-1dfcfdc
+        api_version:
+          type: string
+          example: 1.0.0-dev
+    RegistrationInput:
+      type: object
+      properties:
+        email:
+          type: string
+          format: email
+          example: bizzy@foo.com
+        password:
+          type: string
+          format: password
+          example: secret123
+        username:
+          type: string
+          example: bizzy
+      description: Registration payload
+  requestBodies:
+    RegistrationInput:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RegistrationInput'
+        required: true
+
+#-------------------------------------------------------
+servers:
+  - description: Default server
+    url: http://{host}:{port}/{version}
+    variables:
+      host:
+        default: 'localhost'
+      port:
+        default: '8080'
+      version:
+        default: v1
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/pcrespov/simcore-webserver/1.0.0
+  - description: Production server
```

Comment:
Will remove for the moment ... but could become handy to have a fake server that responds mock data and this is what this swaggerhub offers. But it would be better to get it under itis instead of under my account